internal/distro/fedora32/distro.go

```diff
@@ -426,7 +426,7 @@ func (t *imageType) fsTabStageOptions(uefi bool) *osbuild.FSTabStageOptions {
 func (t *imageType) grub2StageOptions(kernelOptions string, kernel *blueprint.KernelCustomization, uefi bool) *osbuild.GRUB2StageOptions {
 	id := uuid.MustParse("76a22bf4-f153-4541-b6c7-0332c0dfaeac")
 
-	if kernel != nil {
+	if kernel != nil && kernel.Append != "" {
```

Comment:
`kernel` can't e `nil` here anymore now, can it? But I guess it does not hurt to check anyway.

Reply:
Yeah, can't be nil, so probably want to refactor the whole thing to not be a pointer at all. But kept it minimal for now.